desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -490,19 +524,54 @@ protected void onWindowVisible() {
     }
 
     public void updateStatusLabel() {
+        DescriptorProviderStatus remoteErrorState = null;
+        boolean remoteExecuteEnable = true;
+
+        for (ComponentBuilder componentBuilder : _analysisJobBuilder.getComponentBuilders()) {
+            ComponentDescriptor<?> descriptor = componentBuilder.getDescriptor();
+            if (descriptor instanceof RemoteTransformerDescriptorImpl) {
```

Comment:
Same remark as before regarding global applicability of the API / interface change.